src/components/forms/riddleUpsertForm/RiddleShareForm.tsx

```diff
@@ -59,7 +59,14 @@ export const RiddleShareForm = ({
 						label="Riddle link"
 						// required
 						InputProps={{
-							endAdornment: <CopyContentButton content={watchLink ?? ''} />,
+							endAdornment: (
+								<CopyContentButton
```

Comment:
Tady mi asi přijde trošku zvláštní, že kopíruji něco jiného, než je v tom inputu. Tedy `/riddle-detail/<watchLink>` bych zobrazil i přímo v tom inputu. :)

Reply:
To je výborný point, hned pushnu fix